example/offload-template/main.cc

```diff
@@ -0,0 +1,45 @@
+//------------------------------- -*- C++ -*- -------------------------------//
+// Copyright Celeritas contributors: see top-level COPYRIGHT file for details
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file example/offload-template/main.cc
+//! \brief Minimal single-threaded Geant4 application with Celeritas offloading
+//---------------------------------------------------------------------------//
+#include <memory>
+#include <FTFP_BERT.hh>
+#include <G4RunManagerFactory.hh>
+
+#include "src/ActionInitialization.hh"
+#include "src/DetectorConstruction.hh"
+
+//---------------------------------------------------------------------------//
+/*!
+ * Geant4-Celeritas single-threaded offloading template.
+ *
+ * See README for details.
+ */
+int main(int argc, char* argv[])
+{
+    if (argc != 1)
+    {
+        // Print help message
+        std::cout << "Usage: " << argv[0] << std::endl;
+        return EXIT_FAILURE;
+    }
+
+    std::unique_ptr<G4RunManager> run_manager;
+    run_manager.reset(
+        G4RunManagerFactory::CreateRunManager(G4RunManagerType::Serial));
```

Comment:
```suggestion
    run_manager.reset(
        G4RunManagerFactory::CreateRunManager(G4RunManagerType::Default));
```

This will make things work irrespective of the Geant4 install having MT support or not.

Reply:
Yes. I just changed the code to be MT, so it is `Default` now.